libfreerdp/core/smartcardlogon.c

```diff
@@ -245,11 +245,18 @@ static BOOL set_info_certificate(SmartcardCertInfo* cert, BYTE* certBytes, DWORD
 #ifndef _WIN32
 static BOOL build_pkinit_args(const rdpSettings* settings, SmartcardCertInfo* scCert)
 {
+#ifdef WITH_P11KIT
+	const char* defaultModule = "libp11-kit.so";
```

Comment:
I would say it should point to the `p11-kit-proxy.so`. Its a symlink to the same so file:
```
$ ll /usr/lib64/p11-kit-proxy.so 
lrwxrwxrwx. 1 root root 15 Nov 15 01:00 /usr/lib64/p11-kit-proxy.so -> libp11-kit.so.0
```
but I would argue that with this name, its more obvious what is it doing and why. Moreover, the proxy module is documented to be used like this, while the so file you use is made for direct linking and usage of the p11-kit API:

https://p11-glue.github.io/p11-glue/p11-kit/manual/sharing.html

```suggestion
	const char* defaultModule = "p11-kit-proxy.so";
```
(the same in the other file setting this path).

OTOH, it might be even better to get this path from pkgconfig with in one place:
```
pkg_get_variable(P11_PROXY_MODULE p11-kit-1 proxy_module)
```

Reply:
Thanks for your review. I have just changed that to `p11-kit-proxy.so`.